RCTMapboxGL/RCTMapboxGL.h

```diff
@@ -47,6 +47,8 @@ extern NSString *const RCTMGLOnUpdateUserLocation;
 
 @property (nonatomic, strong) UIButton *rightCalloutAccessory;
 @property (nonatomic) NSString *id;
+@property (nonatomic) NSString *annotationImageURL;
+@property (nonatomic) CGSize *annotationImageSize;
```

Comment:
CGSize is a struct, not a class, so you almost never want a pointer to it. You can treat structs like integers, booleans, and other primitives. Delete the asterisk.